reconcile/terraform_resources.py

```diff
@@ -716,3 +721,25 @@ def run(
         cleanup_and_exit(tf, err)
 
     cleanup_and_exit(tf)
+
+
+def early_exit_desired_state(*args, **kwargs) -> dict[str, Any]:
+    gqlapi = gql.get_api()
+    namespaces = get_tf_namespaces()
+    resources = []
+    for ns_info in namespaces:
+        for spec in get_external_resource_specs(
+            ns_info, provision_provider=PROVIDER_AWS
+        ):
+            defaults = spec.resource.get("defaults")
```

Comment:
SQS and dynamodb have their defaults nested

Reply:
good catch! https://github.com/app-sre/qontract-reconcile/pull/2714/commits/3a50e9c0509c30753d27eab8100e2c3e89a4494d